otter/test/models/test_cass_models.py

```diff
@@ -2527,6 +2557,68 @@ def test_add_scaling_policy_cron(self):
         self.assertEqual(result, [pol])
 
 
+class ScalingGroupWebhookMigrateTests(SynchronousTestCase):
+    """
+    Tests for webhook migration functions in :obj:`CassScalingGroup`
+    """
+
+    def setUp(self):
+        """
+        Sample CassScalingGroup object
+        """
+        self.store = CassScalingGroupCollection(None, None)
+
+    def test_webhook_index_only(self):
+        """
+        Test `get_webhook_index_only`
```

Comment:
What aspect of `get_webhook_index_only` is this testing?  `"w2": "2"` is not the most self-documenting of examples :).